src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/helpers.js

```diff
@@ -27,6 +27,10 @@ export const processBlock = _block => {
   const block = _block;
```

Comment:
should be `const block = { ..._block }` as the way it stands code further down will mutate the `_block` that is passed in

Reply:
Cheers Jroe, done